packages/apps/human-app/server/src/modules/cron-job/cron-job.service.ts

```diff
@@ -48,9 +48,11 @@ export class CronJobService {
     this.logger.log('CRON START');
 
     const oracleDiscoveryCommand: OracleDiscoveryCommand = {};
-    const oracles = await this.oracleDiscoveryService.processOracleDiscovery(
-      oracleDiscoveryCommand,
-    );
+    const oracles = (
+      await this.oracleDiscoveryService.processOracleDiscovery(
+        oracleDiscoveryCommand,
+      )
+    )?.oracles;
```

Comment:
Do we need `?` here? According to types the result can't be null

Reply:
removed